Dockerfile

```diff
@@ -84,7 +85,7 @@ RUN ln -s ${HADOOP_HOME}/share/hadoop/tools/lib/*aws* $HIVE_HOME/lib
 # Configure MySQL connector jar to be available to hive
 RUN ln -s /usr/share/java/mysql-connector-java.jar "$HIVE_HOME/lib/mysql-connector-java.jar"
 # Configure Postgesql connector jar to be available to hive
-RUN ln -s /usr/share/java/postgresql-jdbc.jar "$HIVE_HOME/lib/postgresql-jdbc.jar"
+RUN ln -s /usr/share/java/postgresql.jar "$HIVE_HOME/lib/postgresql-jdbc.jar"
```

Comment:
I haven't dived into the image build logs, but I do see that check has passed. That said, looking at original copy instruction (line 76):

```
COPY --from=build /build/postgresql.jar /usr/share/java/postgresql-jdbc.jar
```

It looks like you're attempting to symbolically link a file that doesn't exist.

Reply:
This file is in build. I stopped the docker build and checked the intermediate container.

Reply:
No, I meant that the destination `/usr/share/java/postgresql-jdbc.jar` path in the container is named `postgresql-jdbc.jar`, but you're attempting to symbolically link `/usr/share/java/postgresql.jar`, which shouldn't exist.

Reply:
Gotcha... I had initially thought it didn't move from bin dir immediately. I know now better. Ty